react/features/app/actions.js

```diff
@@ -141,6 +144,8 @@ export function appNavigate(uri: ?string) {
         if (room && navigator.product === 'ReactNative') {
             dispatch(createDesiredLocalTracks());
             dispatch(connect());
+            navigate(screen.conference.main, {
```

Comment:
Ditto. Also, why do we need to specify the screen twice?

Reply:
Even if we merged the two navigators, the conference screen is still in another one.